### PR TITLE
Sort languages and make pie chart more colorful

### PR DESCRIPTION
**Pull request by colbyguan**
I thought it was a little hard to read the chart since some colors would be re-used. The sorting should also make it easier to see a user's most preferred languages. The added dependency is ~1kB, hopefully that's okay.

Before:
image

After:
image

### CODE REVIEW

**Comment by theblindprophet** [comment]
I like the sorting, but I think the colors are going to clash with the rest of the sight. If you want to spend some time getting more colors that are brighter, but still match the theme that would be very appreciated.

**Comment by theblindprophet** [comment]
I would rather keep the same color scheme, or close to it. This color set would be cool if is was our theme colors.
screen shot 2018-10-02 at 9 32 02 pm

**Comment by colbyguan** [comment]
I have taken those 4 columns, rows 100-900 and dropped those into a `constants.js` file to select from, some sample results:

image
image

File: src/components/Home-Languages.js [hunk 1]
```diff
@@ -62,4 +62,16 @@ class HomeLanguages extends Component {
   }
 }
 
+// Java String hashcode implementation
```
Comment (theblindprophet) [inline src/components/Home-Languages.js]: This needs to be inside the class and follow es6 format. e.g. `simpleHash = s => {...}`

**Comment by theblindprophet** [comment]
Also, use these colors. its a good mix that matches the site.

```
#9ad2dd
#9699ac
#8dccd8
#389caf
#8c90a5
#86cbd8
#81c6d4
#83879d
#74c0d0
#797d96
#67bbcb
#70748e
#6cc0d0
#5bb5c7
#686c84
#3490a2
#4eafc3
#61657b
#41a9bf
#595d71
#3d9fb2
#525568
#3893a6
#308595
#4a4d5e
#348899
#434655
#307d8c
#3ca7bc
#2b7180
#93d0dc
#276673
#79c5d4
#235b66
```

**Comment by colbyguan** [comment]
@theblindprophet rebased and addressed comments!

**Comment by theblindprophet** [comment]
@colbyguan Great work on this, I made some very minor changes and have moved the PR to #55.

**Pull request closed without merging**